resources/views/forms/show.blade.php

```diff
@@ -30,11 +30,23 @@
                 @endcan
             </dropdown-list>
 
+            @php
+                $exporters = collect(config('statamic.forms.exporters', []))->filter(function ($exporter) use ($form) {
+                    if (! $limits = $exporter['forms'] ?? []) {
+                        return true;
+                    }
+
+                    return in_array($form->handle(), $limits);
+                })->all();
+            @endphp
+            @if (count($exporters))
             <dropdown-list>
                 <button class="btn" slot="trigger">{{ __('Export Submissions') }}</button>
-                <dropdown-item :text="__('Export as CSV')" redirect="{{ cp_route('forms.export', ['type' => 'csv', 'form' => $form->handle()]) }}?download=true"></dropdown-item>
-                <dropdown-item :text="__('Export as JSON')" redirect="{{ cp_route('forms.export', ['type' => 'json', 'form' => $form->handle()]) }}?download=true"></dropdown-item>
+                @foreach ($exporters as $type => $exporter)
+                <dropdown-item :text="__('{{ $exporter['label'] ?? __('Export as '.strtoupper($type)) }}')" redirect="{{ cp_route('forms.export', ['type' => $type, 'form' => $form->handle()]) }}?download=true"></dropdown-item>
```

Comment:
Let's change the label to `text="{{ $exporter['class']::title() }}"`

The `title()` method should have a `__()` function used, like in Fieldtype.php.

I don't think they all need to say `Export as`.

Reply:
Thats updated